public/styles/main.css

```diff
@@ -0,0 +1,67 @@
+/* http://meyerweb.com/eric/tools/css/reset/ 
+   v2.0 | 20110126
+   License: none (public domain)
+*/
+
+html, body, div, span, applet, object, iframe,
```

Comment:
What's the purpose of setting these?  They look like the default values if you weren't going to specify them, is it something to do with styled components?